surfactant/infoextractors/native_lib_file.py

```diff
@@ -96,7 +96,7 @@ def match_by_attribute(attribute: str, content: str, patterns_database: Dict) ->
                 if attribute == "filename":
                     if name == content:
```

Comment:
```suggestion
                    if name.lower() == content.lower():
```

Let's make this a case-insensitive comparison.